### PR TITLE
Two PIP cards

### PR DESCRIPTION
**Pull request by dracontes**
- [Young Deathclaws](https://scryfall.com/card/pip/125/young-deathclaws)
- [Yes Man, Personal Securitron](https://scryfall.com/card/pip/29/yes-man-personal-securitron): While I was working on it, it wasn't showing the `StackDescription$` text I placed on the reflexive trigger. Hopefully setting it to `None` fixes things satisfactorily, though I understand that `StackDescription$` has current general issues being displayed correctly.

### CODE REVIEW

File: forge-gui/res/cardsfolder/upcoming/young_deathclaws.txt [hunk 1]
```diff
@@ -0,0 +1,7 @@
+Name:Young Deathclaws
+ManaCost:2 B G
+Types:Creature Lizard Mutant
+PT:4/2
+K:Menace
+S:Mode$ Continuous | Affected$ Creature.YouOwn | AffectedZone$ Graveyard | AddKeyword$ Scavenge:CardManaCost | Description$ Each creature card in your graveyard has scavenge. The scavenge cost is equal to its mana cost.
```
Comment (Simisays) [inline forge-gui/res/cardsfolder/upcoming/young_deathclaws.txt:6]: Maybe deckhints: Ability$Graveyard|Mill

Comment (dracontes) [reply forge-gui/res/cardsfolder/upcoming/young_deathclaws.txt:6]: Oh, sure thing :) While I was at it, I had a look at other scavenge cards and lifted pointers from them in this regard.

File: forge-gui/res/cardsfolder/upcoming/yes_man_personal_securitron.txt [hunk 2]
```diff
@@ -0,0 +1,14 @@
+Name:Yes Man, Personal Securitron
+ManaCost:2 W
+Types:Legendary Artifact Creature Robot
+PT:2/2
+A:AB$ GainControl | Cost$ T | Defined$ Self | ValidTgts$ Opponent | TgtPrompt$ Select target opponent | SubAbility$ DBImmediateTrigger | PlayerTurn$ True | StackDescription$ SpellDescription | SpellDescription$ Target opponent gains control of CARDNAME. When they do, you draw two cards and put a quest counter on NICKNAME.
+SVar:DBImmediateTrigger:DB$ ImmediateTrigger | ConditionDefined$ Self | ConditionPresent$ Card.ControlledBy ParentTarget | Execute$ DBDraw | StackDescription$ None
```
Comment (Hanmac) [inline forge-gui/res/cardsfolder/upcoming/yes_man_personal_securitron.txt:6]: `TriggerDescription$ When they do, you draw two cards and put a quest counter on NICKNAME.`

File: forge-gui/res/cardsfolder/upcoming/yes_man_personal_securitron.txt [hunk 3]
```diff
@@ -0,0 +1,14 @@
+Name:Yes Man, Personal Securitron
+ManaCost:2 W
+Types:Legendary Artifact Creature Robot
+PT:2/2
+A:AB$ GainControl | Cost$ T | Defined$ Self | ValidTgts$ Opponent | TgtPrompt$ Select target opponent | SubAbility$ DBImmediateTrigger | PlayerTurn$ True | StackDescription$ SpellDescription | SpellDescription$ Target opponent gains control of CARDNAME. When they do, you draw two cards and put a quest counter on NICKNAME.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/yes_man_personal_securitron.txt:5]: description incomplete

File: forge-gui/res/cardsfolder/upcoming/yes_man_personal_securitron.txt [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+SVar:DBImmediateTrigger:DB$ ImmediateTrigger | ConditionDefined$ Self | ConditionPresent$ Card.ControlledBy ParentTarget | Execute$ DBDraw | TriggerDescription$ When they do, you draw two cards and put a quest counter on NICKNAME.
+SVar:DBDraw:DB$ Draw | NumCards$ 2 | SubAbility$ DBCounter
+SVar:DBCounter:DB$ PutCounter | Defined$ Self | CounterType$ QUEST | CounterNum$ 1
+T:Mode$ ChangesZone | Origin$ Battlefield | Destination$ Any | ValidCard$ Card.Self | Execute$ TrigToken | TriggerDescription$ Wild Card — When NICKNAME leaves the battlefield, its owner creates a tapped 1/1 white Soldier creature token for each quest counter on it.
+SVar:TrigToken:DB$ Token | TokenAmount$ X | TokenScript$ w_1_1_soldier | TokenOwner$ TriggeredCardOwner | TokenTapped$ True | SubAbility$ DBCleanup
+SVar:DBCleanup:DB$ Cleanup | ClearRemembered$ True
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/yes_man_personal_securitron.txt:11]: what are you cleaning up?

Comment (dracontes) [reply forge-gui/res/cardsfolder/upcoming/yes_man_personal_securitron.txt]: Oops, forgot to remove that. Had some trouble making sure the tokens showed up on the right side of the battlefield and iterated a few approaches gleaned from existing scripts.

**Review by tool4ever — APPROVED** [review]
thanks

**Pull request merged**